roles/openshift_logging/defaults/main.yml

```diff
@@ -13,10 +13,26 @@ openshift_logging_curator_run_minute: 0
 openshift_logging_curator_run_timezone: UTC
 openshift_logging_curator_script_log_level: INFO
 openshift_logging_curator_log_level: ERROR
+openshift_logging_curator_cpu_limit: 100m
+openshift_logging_curator_memory_limit: null
```

Comment:
Agree with alternate point, although we also have it in the docs to cover that.
Are our templates not checking if the vars are defined as part of the check if we should be adding the section?